src/components/my-sale-card.js

```diff
@@ -104,7 +104,7 @@ class MySaleCard extends Component {
               </h2>
               <p className="address text-muted">{user.address}</p>
               <div className="d-flex">
-                <p className="price">
+                <p className="price" data-createdAt={ createdAt }>
```

Comment:
@wanderingstan why dis? 😕

Reply:
Linter bitches about `createdAt` not being used. My limited javascript and a quick search couldn't find a way to throw away the value retrieved for `createdAt` at line 53. Do you know a way? Figured we might want to show the created date at some other point, so no harm in putting it in a element prop.

Reply:
Well, the short answer is just to [remove it](https://github.com/OriginProtocol/origin-dapp/commit/53484a4385c28931e0c2410bf2aff7ab47a96fab) if it's not being used. But it begs the question whether or not we're supposed to be showing a timestamp on that cart. I'll confirm.